libs/webb-ui-components/src/components/ConnectWalletMobileButton/ConnectWalletMobileButton.tsx

```diff
@@ -38,24 +45,44 @@ export const ConnectWalletMobileButton = forwardRef<
         </Button>
       </BottomDialogTrigger>
       <BottomDialogPortal
-        title="Try Hubble on Desktop"
-        actionButtonsProps={actionButtonsProps}
+        title={
+          appType === 'bridge-dapp'
+            ? 'Try Hubble on Desktop'
+            : 'Switch to Desktop'
+        }
```

Comment:
Pass the `title` as a prop to this component and use the existing value as the default title.